cgo_static.go

```diff
@@ -4,7 +4,7 @@ package duckdb_go_extensions
 
 ////-lsubstrait_extension
 /*
-#cgo LDFLAGS: -licu_extension -ltpch_extension -ltpcds_extension -lsubstrait_extension
+#cgo LDFLAGS: -licu_extension -ltpch_extension -ltpcds_extension -lsubstrait_extension -lparquet_extension -lduckdb_mbedtls
```

Comment:
This used to cause the build to fail (specifically parquet) as it was already linked. Has upstream changed such that is not true anymore? Also, why the addition of mbedtls?